.github/workflows/coordinator-build-and-publish.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         uses: gradle/actions/setup-gradle@cc4fc85e6b35bafd578d5ffbc76a5518407e1af0 #v4.2.1
```

Comment:
Not a focus of this PR, but I think it was released as a part of v4 release

Reply:
I don't get your point/suggestion... can you elaborate please?

Reply:
To leverage changes in this commit we can just use a regular import like 'gradle/actions/setup-gradle@v4' instead of this cryptic one

Reply:
That that was the approach we had before. It was suggested by ops team because the V4 is an alias to the latest v4 and had regressions. With static approach it we have confidence it won't have regressions